plotsquared/schematics/road-schematic.md

```diff
@@ -0,0 +1,57 @@
+# Plotworld Road Schematic
+
+## Introduction
+
+PlotSquared allow to customize **the road between the plots** with predefined schematics.
+
+{% hint style="tip" %}
+The road-schematics only affect the Road of the Plot world. Not the plot itself. For instructions on how to set up plot-schematics, see the articles [Plot Schematic on Generation](../schematics/schematic-generation.md) and [Plot Schematic on Claim](../schematics/schematic-on-claim.md).
+{% endhint %}
+
+{% hint style="info" %}
+Road schematics can be added after world generation. You are able to change the road schematic whenever you want, however, this will only affect [newly generation regions](#4-world-regeneration) and does not change previously generated regions.
```

Comment:
```suggestion
Road schematics can be added after world generation. You are able to change the road schematic whenever you want. However, this will only affect [newly generation regions](#4-world-regeneration) and does not change previously generated regions.
```